<!--<div class="container"> -->
<div class="container-fluid"> 
	<div class="col-md-10 col-md-offset-1">
		<!--<div class="row"> -->
			<br>
			<!--<div class="col-md-10"> -->
			<p class="lead"><strong>About us</strong></p>
			<hr style="border:2px solid black">
			<table class="table table-striped">
				<tbody>
					<!--<tr><th>Department</th><td>Center for Medical Genetics & Hunan Key Laboratory of Medical Genetics, School of Life Sciences, Central South University (CMG/HKLMG-SLC-CSU)<br> Peking University Third Hospital (PUTH)</td></tr>
					<tr><th>Address</th><td>CMG/HKLMG-SLC-CSU: 72 Xiangya Road, Changsha 410008, Hunan, China <br> BIOLS: 49 Huayuan North Road, Haidian District, Beijing 100191, China.</td></tr> -->
					<tr><th>Department</th><td>Peking University Third Hospital (PUTH)</td></tr>
					<tr><th>Address</th><td>49 Huayuan North Road, Haidian District, Beijing 100191, China.</td></tr>
					<tr><th>Research</th><td>Genome Medicine, Genomics and Epigenomics, Bioinformatics.</td></tr>
				</tbody>
			</table>
			<p align="justify">Our team is dedicated to complex disease studies, focusing on development of biological databases by large-scale data integration and data mining, and development of bioinformatics tools and web servers to facilitate researchers to analyze the epigenomic, genetic and regulatory data of complex diseases. We mainly focus on interpretation of genetic variations and epigenetic regulations in human cancers and psychiatric disorders. 
				Our published disease related databases include a functional variants database involved in post-transcriptional regulation mediated by RNA-binding 
				(<a href="http://www.rbp-var.biols.ac.cn/" class="input_example">RBP-Var</a>, 2016), 
				a platform for linking regulatory <I> de novo </I> mutations to developmental epigenetics and diseases 
				(<a href="http://www.epidenovo.biols.ac.cn/" class="input_example">EpiDenovo</a>, 2018),
				a platform for decoding regulatory somatic mutations in human cancers 
				(<a href="http://www.oncobase.biols.ac.cn/" class="input_example">OncoBase</a>, 2019) and
				an integrated database and analysis platform for oncogenic driver variants in human cancers
				(<a href="https://oncovar.org/" class="input_example">OncoVar</a>, 2021).
				Our published bioinformatics tools include a Bayesian framework to identify methylcytosines from high-throughput bisulfite sequencing data
				(<a href="https://github.com/maofengbiao/Bycom" class="input_example">Bycom</a>, 2014), 
				estimating genome-wide single-base methylation level from MB-seq data combined with RRBS data 
				(<a href="https://github.com/maofengbiao/MBRidge" class="input_example">MBRidge</a>, 2015),
				a deep learning algorithm for identifying posttranscriptionally impaired SNVs
				(<a href="https://github.com/WeiWenqing/PIVar" class="input_example">PIVar</a>, 2020) and 
				an ensemble learning method for identifying driver mutations in personal cancer genomes
				(<a href="https://github.com/hatchetProject/AI-Driver" class="input_example">AI-Driver</a>, 2020).
				</p>
				<!--
							Our lab is broadly interested in understanding the genetic basis of complex human diseases using next-generation sequencing (NGS). 
							We develop and employ computational or statistical tools, web servers and databases to prioritize candidate genes and decipher the relationship between genotypes
							and phenotypes from sequencing and clinical data. We mainly focus on neuropsychiatric disorders, such as autism. 
							We also employed molecular biology and neuroscience to study the pathogenesis of candidate genes in human genetic diseases. 
							Our published disease related databases include <a href="http://centre.bioinformatics.zj.cn/EpilepsyGene" target="_blank">EpilepsyGene</a>, 2014,
							a genetic resource for genes and mutations related to epilepsy; <a href="http://www.wzgenomics.cn/NPdenovo/index.php" target="_blank">NPdenovo</a>, 2015,
							a database of de novo mutation in neuropsychiatric disorders; <a href="http://www.rbp-var.biols.ac.cn/" target="_blank">RBP-Var</a>, 2016, a functional variants database involved 
							in post-transcriptional regulation mediated by RNA-binding. Our published web servers for NGS data analysis include 
							<a href="http://centre.bioinformatics.zj.cn/mirtools/" target="_blank">mirTools</a>, 2010 for microRNA profilling and discovery; 
							<a href="http://centre.bioinformatics.zj.cn/mr2_dev"  target="_blank">mirTools 2.0</a>, 2013 for non-coding RNA discovery, profiling, and functional annotation; and 
							<a href="http://www.wzgenomics.cn/mirTrios/" target="_blank">mirTrios</a>, 2014 for de novo mutation detection. 
							Our published bioinformatics tools include <a href="http://bioinformatics.zj.cn/magicviewer/"  target="_blank">MagicViewer</a>, 2010 for NGS data visualization,
							genetic variation detection and annotation; 
							<a href="http://sourceforge.net/projects/bycom/" target="_blank">Bycom</a>, 2014 for methylcytosine calling from BS-seq data;
							<a href="http://sourceforge.net/projects/mbridge/"  target="_blank">MBRidge</a>, 2015 for profiling DNA methylome at single-base resolution;
							and <a href="https://github.com/xflicsu/swDMR"  target="_blank">swDMR</a>, 2015 for differentially methylated region identification.</p>
							
				-->
				<br/>
				<p class="lead"><strong>CircleBase team</strong></p>
				<hr style="border:2px solid black">
				<table class="table table-striped">
					<thead>
						<tr>
							<th>Name</th>
							<th>E-mail</th>
						</tr>
					</thead>
					<tbody>
						<tr><td>Xiaolu Zhao</td><td>xiaolu_zhao@163.com</td></tr>
						<tr><td>Leisheng Shi</td><td>slsbioinfo@126.com</td></tr>
						<tr><td>Mingkun Li</td><td>limk@big.ac.cn</td></tr>
						<tr><td>Fengbiao Mao</td><td>maofengbiao@126.com</td></tr>
					</tbody>
				</table>
			
			<p class="lead"><strong>Feedback</strong></p>
			<hr style="border:2px solid black">
			<p>We hope you enjoy this database as much as we do and please feel free to email us with any suggestions or questions.</p>
			<p>For bug report, email to <strong>slsbioinfo@126.com</strong>. </p>
			<p>For data source recommendation, email to <strong>maofengbiao@126.com</strong>.</p>
			</div>
		<!--</div> -->
	<!--</div>-->
</div>
